Project.toml

```diff
@@ -52,7 +52,7 @@ IsApprox = "0.1"
 KrylovKit = "0.4.2, 0.5, 0.6, 0.7"
 LinearAlgebra = "1.6"
 LinearMaps = "3"
-NDTensors = "0.2.30"
+NDTensors = "0.2.31"
```

Comment:
```suggestion
NDTensors = "0.3"
```

Reply:
Oh yeah I didn't realize this was the convention. Thank you for the explanation!